css/custom.css

```diff
@@ -0,0 +1,9 @@
+h1, h2, h3, h4, h5, h6 {
```

Comment:
Man atrodo nusimus visu Headingai, jeigu taip padarysi, gal geriau atskirai savo faile pasidaryk?

Reply:
tik del to kad visame puslapyje kartojasi sitie stiliai, gal uppercase galima isimti